packages/dcos-image-deps/build

```diff
@@ -5,8 +5,8 @@ mkdir -p "$LIB_INSTALL_DIR"
 
 # Install wheels.
 for package in adal analytics-python azure-nspkg azure-common azure-mgmt-nspkg \
-    azure-mgmt-network azure-storage beautifulsoup4 docutils keyring msrest \
-    msrestazure pytest-xdist requests-oauthlib schema webob wheel; do
```

Comment:
Hi @cprovencher 

In this file we are installing only `execnet` while on dcos-integration-test/requirements.txt file, we install `pytest-xdist` - Would it create any problems?

Thanks!